docker/mattermost/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+fastapi = "*"
```

Comment:
can you check if demisto/fastapi works for you?

Reply:
![image](https://github.com/demisto/dockerfiles/assets/93524335/60a28e48-2204-4319-87d4-41e0b536f186)

Reply:
so lets add what is missing to demisto/netutils and use that